### DISCUSSION TITLE
DOMSubtreeModified No longer works in Chrome

### DISCUSSION THREAD

**Discussion by nrdevau** [jquery/jquery#5535] [General]
So, I've just inherited a codebase running an older version of Jquery (3.6.0) in which case some code I need to stabilise has layer up layers (requirejs -> jquery ui -> jquery) usage of `.click` instead of `.on` usage causing some functionality to stop working on the site, only in Chrome.

The state of the codebase is so bad (6 generation of agency pirates) causing the amount of errors in the console being unmanagable, so I've been filtering out the jqmigrate errors, to my detriment it would seem.

I have setup a JSFiddle to demonstrate this at its core. Running the fiddle in the current Chrome and the current FF shows different results.

I can't imagine I'm the only one affected by this, so I'd like to start a discussion to at least capture the class of errors people in the wild are dealing with.

Here's the Fiddle: https://jsfiddle.net/nrdev/kwLu2yvh/5/

My current gameplan is:
1. Replace calls to `.click` with `.on('click'` and see what else is breaking
2. Eventually look at migrating Jquery versions
3. No longer have this issue?

Just wanted to rant really, I was going to make an issue but it said not to raise issues for older versions, so I figured this is the next best thing :+1:

**Comment by nrdevau** [comment]
The current version of jQuery is only one minor version ahead.. 3.7.1 from what I can see here (https://releases.jquery.com/jquery/)

Hopefully the migration path isn't too bumpy. Good luck everyone else!

**Comment by mgol** [comment]
The `.click` alias is just deprecated, not removed. We recommend migrating away from it, but its usage shouldn’t break anything at this point.

**Comment by nrdevau** [comment]
Ah yes, thanks @mgol looks like the `.click` deprecation warning was just a red herring. The point of failure in my code is relying on the DOMSubtreeModified event.
In my specific situation, there's a 3rd party tool that appends elements to my site, with which the site has custom code to then manipulate what the 3rd party is doing. I'll be converting my code from relying on that event to using a `MutationObserver`

Maybe someone else out there will have a similar issue and see this.

Fun times.

**Comment by nrdevau** [comment]
For anyone playing along at home: https://jsfiddle.net/nrdev/kwLu2yvh/8/ is working in FF but not Chrome. Working on a migration demo

**Comment by nrdevau** [comment]
It gets better! https://jsfiddle.net/nrdev/kwLu2yvh/16/
Now we have new race conditions to make sure we code around :+1: Wooo!

**Comment by nrdevau** [comment]
I'm going to close this as it's not jQuery's fault. Keep on keeping on
